ui/tract-renderer/src/main/java/org/cru/godtools/tract/viewmodel/PageViewHolder.java

```diff
@@ -116,6 +116,9 @@ void onVisible() {
         } else {
             mHeroViewHolder.markVisible();
         }
+        if (!mPageContentLayout.isBounceFeatureDiscovered()) {
+            mPageContentLayout.setAnimateCard(true);
```

Comment:
I'm going to rework this logic to enable/disable the bounce on the PCL. I also want to monitor for the card feature to be discovered here.

Reply:
and I have an idea of one more thing to do with feature discovery :)